.github/CODEOWNERS

```diff
@@ -1 +1,6 @@
-*       @kolenaIO/kolena-owners
+*                   @kolenaIO/kolena-owners
+
+README.md           @kolenaIO/docs-owners
+mkdocs.yml          @kolenaIO/docs-owners
+mkdocs.insiders.yml @kolenaIO/docs-owners
+docs/*              @kolenaIO/docs-owners
```

Comment:
@munkyshi @ly-kolena thoughts on this?

Reply:
Depends on who would be assigned docs-owners and how we imagine their responsibilities, from my thinking, I'd propose to just assign `docs/*` to docs-owners, who would be responsible for reviewing docs content.

Otherwise looks good to me.

Reply:
I’ve created the group and added us, @munkyshi, and @y27choi. The README and other files are related to documentation — almost every docs PR will update the nav in `mkdocs.yml`.